src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.js

```diff
@@ -1,5 +1,51 @@
-import React from 'react'
+import React, { Component } from 'react'
+import './ActivityHistory.css'
+import { fetchGoalHistory } from '../../../../goal-history/actions'
+import { connect } from 'react-redux'
 
-export default function ActivityHistory() {
-  return <h1>Historial</h1>
+class ActivityHistory extends Component {
+  render() {
+    this.props.fetchGoalHistory({
+      address: this.props.goalAddress,
+    })
+    console.log('props.state', this.props.state)
+    return (
+      <div className='history'>
+        <img src='#' className='historyPic' alt='user Pic' />
+        <div className='historyBody'>
+          <div className='historyHeader'>
+            <span className='historyDate'>fecha</span>
+          </div>
+          <div style={{ display: 'flex' }}>
+            <h3
+              className='historyAuthor'
+              style={{ margin: '0', marginRight: '15px' }}>
+              Nombre user
+            </h3>
+            <span className='historyContent'>comentario</span>
+          </div>
+        </div>
+      </div>
+    )
+  }
 }
+
+function mapStateToProps(state) {
+  const goalAddress = state.ui.expandedView.goalAddress
+  return {
+    goalAddress,
+    avatarAddress: state.whoami.entry.address,
+    avatarUrl: state.whoami.entry.avatar_url,
+    agents: state.agents,
+    state: state,
```

Comment:
this might have just been for development purposes, but don't pass in the whole state object